stdlib/public/core/UnicodeScalarProperties.swift

```diff
@@ -519,35 +519,35 @@ extension Unicode.Scalar.Properties {
     return _hasBinaryProperty(__swift_stdlib_UCHAR_CASE_IGNORABLE)
   }
 
-  /// A Boolean property indicating whether the scalar is one whose normalized
-  /// form is not stable under a `toLowercase` mapping.
+  /// A Boolean property indicating whether the scalar's `lowercaseMapping`
+  /// differs from the NFD normalized form of the scalar itself.
```

Comment:
This description is not quite accurate. The point about NFD is more of a specification detail, since NFD is simpler to specify properties in terms of and maintains its own normality under these mappings. But, I can't find any guarantee that `lowercaseMapping` produces NFD.

Also, since normalization may change one scalar into many, the spec's mention of case mapping is as applied to the resulting sequence of scalars, so it's a little odd to talk in terms of `lowercaseMapping` on the scalar directly.

But, we may be able to infer (and we should verify) that `x.changesWhenLowercased == (String(x) == x.lowercaseMapping)`, as this property exists to optimize case logic fast-paths.

Strawman: "A Boolean property indicating whether the scalar's normalized form differs from the `lowercaseMapping` of each constituent scalar."

Though I find "each constituent scalar" a little hard to read...

Reply:
I'll defer to your suggestion since upon re-reading them both, yours is truer to §3.13. I tried thinking of other ways to phrase "each constituent scalar" but couldn't land on anything I liked better after about 5 minutes, so let's go with that.